### ISSUE TITLE
Update make.bat to match Makefile

### ISSUE THREAD

**Issue by tcmetzger** [bokeh/bokeh#12480]
After #12339 and some other follow-up PRs, we need to update the docs `make.bat` to match the new `Makefile`.
Currently, running `make.bat` fails with several errors, mostly due to outdated paths, it seems.

**Comment by chinmaychahar** [comment]
Hello @tcmetzger I faced these issues while running `make html` for setting up local documentation on my windows machine. I believe I'll be able to work on this issue and update the outdates paths wherever needed. Can I work on this?

**Comment by AnishereMariam** [comment]
@tcmetzger, I tried running the `make.bat` command and realized that it returned the same error I got from building docs with `sphinx-build` #12548. Error Code: `configparser.MissingSectionHeaderError`. 

I would love to get your opinion on how to resolve this error so that we can try to solve both problems at once.

**Comment by tcmetzger** [comment]
@chinmaychahar and @anisheremariam Thank you for looking into this! I just tried running make.bat on win10 again, and I did get errors, most prominently `Error generating bokeh-content-248572d9a7fb4b509bc09a07c86f10a5-docs-examples-advanced-extensions-putting_together.js:`. I did not get the `configparser` error you mentioned (running sphinx-build.exe directly gets the same result). I don't usually use windows, so I'm afraid I can't be much help... For #12548, using a virtual machine or WSL is probably the best idea. For this issue here, it might be helpful to post your console output. This is the output I get on win10:

```
Sphinx error:
Error generating bokeh-content-9a96432a175d4e1882710de74d1f0882-docs-examples-advanced-extensions-putting_together.js:

...

Error: Cannot find module 'eslint/use-at-your-own-risk'
```

**Comment by chinmaychahar** [comment]
@tcmetzger I actually encountered the same error that you've mentioned above. Posting the output below for your reference:

```reading sources... [  2%] docs/examples/advanced/extensions/putting_together
Sphinx error:
Error generating bokeh-content-46e1727a9b2b458bba5668225f55251e-docs-examples-advanced-extensions-putting_together.js:

C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:91
          throw err;
          ^

Error: Cannot find module 'eslint/use-at-your-own-risk'
    at require (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:89:23)
    at C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:378688:9
    at Object._ (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:378516:64)
    at require (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:107:21)
    at Object._ (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:348952:45)
    at require (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:107:21)
    at Object._ (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:348582:37)
    at require (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:107:21)
    at Object._ (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:182902:5)
    at require (C:\Windows\System32\bokeh\src\bokeh\server\static\js\compiler.js:107:21) {
  code: 'MODULE_NOT_FOUND'
}

Node.js v18.12.1

File not found - static
0 File(s) copied

**Comment by chinmaychahar** [comment]
I tried skimming through issue #12119 and get some ideas on how to resolve it but count not find any work around

**Comment by bryevdv** [comment]
@chinmaychahar would you be able to try with docker? 

TBH given the lack of depth we have on the windows side I think we are really going to just have no choice but to more heavily promote using a docker env for development, just to reduce the amount of variability to a manageable level. I just don't know how to fix or address these problems on windows 😞

**Comment by chinmaychahar** [comment]
> @chinmaychahar would you be able to try with docker?

Okay, I will try to run it with docker. Any guides I could follow for that would be helpful, thanks!

**Comment by tcmetzger** [comment]
@chinmaychahar I don't think the docker env is available to develop docs on the 3.2 branch yet (@ianthomas23 knows the details). I would suggest using Bokeh with WSL2 on Windows - that is how I do all work on Bokeh on Windows these days. I have also worked on the docs with VirtualBox/Ubuntu!

**Comment by mattpap** [comment]
> `Error: Cannot find module 'eslint/use-at-your-own-risk'`

This has been causing problems for awhile now. It's curious though why it works on Linux and not Windows. Looking at compiler's bundle, this should fail on both platforms.

**Comment by ianthomas23** [comment]
I wouldn't recommend using the docker dev image until we've ironed out the problems running it on Windows and updated it for `branch-3.2`.

**Comment by tcmetzger** [comment]
@mattpap @bryevdv as discussed, I have cloned a fresh copy of the repo on Win10 and set everything up as described in our contributor guide. When I run `make.bat html`, the following things happen (tested on two separate Win10 machines and with both branch-3.4 and branch-3.3 - with the same outcome):

```
(bkdev) C:\github\bokeh\docs\bokeh>make.bat html
Running Sphinx v7.2.6
C:\github\bokeh\src\bokeh\sphinxext\bokeh_gallery.py:44: RemovedInSphinx80Warning: The alias 'sphinx.util.status_iterator' is deprecated, use 'sphinx.util.display.status_iterator' instead. Check CHANGES for Sphinx API modifications.
  from sphinx.util import ensuredir, status_iterator
C:\github\bokeh\src\bokeh\sphinxext\bokeh_plot.py:104: RemovedInSphinx80Warning: The alias 'sphinx.util.status_iterator' is deprecated, use 'sphinx.util.display.status_iterator' instead. Check CHANGES for Sphinx API modifications.
  from sphinx.util import copyfile, ensuredir, status_iterator
C:\github\bokeh\src\bokeh\sphinxext\bokeh_sitemap.py:41: RemovedInSphinx80Warning: The alias 'sphinx.util.status_iterator' is deprecated, use 'sphinx.util.display.status_iterator' instead. Check CHANGES for Sphinx API modifications.
  from sphinx.util import status_iterator
making output directory... done
creating gallery file entries... [100%] missing_dates.rst
[autosummary] generating autosummary for: docs/dev_guide.rst, docs/dev_guide/bindings.rst, docs/dev_guide/bokehjs.rst, docs/dev_guide/creating_issues.rst, docs/dev_guide/debug_headless_chrome.rst, docs/dev_guide/documentation.rst, docs/dev_guide/env_vars.rst, docs/dev_guide/models.rst, docs/dev_guide/pull_requests.rst, docs/dev_guide/python.rst, ..., docs/user_guide/topics/contour.rst, docs/user_guide/topics/geo.rst, docs/user_guide/topics/graph.rst, docs/user_guide/topics/hex.rst, docs/user_guide/topics/hierarchical.rst, docs/user_guide/topics/images.rst, docs/user_guide/topics/pie.rst, docs/user_guide/topics/stats.rst, docs/user_guide/topics/timeseries.rst, index.rst
BokehDeprecationWarning: 'tile_providers module' was deprecated in Bokeh 3.0.0 and will be removed, use 'add_tile directly' instead.
loading intersphinx inventory from https://numpy.org/doc/stable/objects.inv...
loading intersphinx inventory from https://pandas.pydata.org/pandas-docs/stable/objects.inv...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://www.sphinx-doc.org/en/master/objects.inv...
loading intersphinx inventory from https://xyzservices.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
writing output...
building [html]: targets for 610 source files that are out of date
updating environment: [new config] 610 added, 0 changed, 0 removed
 * Serving Flask app 'bokeh_app_9b25ccc505b0478d9f5ba38c8c3ac53a'
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5050
Press CTRL+C to quit
```
-> The docs begin to build but at some point the server started (I only called `make.bat html`, NOT  `make.bat serve`). When I open the URL in my browser, I only see a 404 error.

Next, I hit Ctrl+C to quit the server. The following output happens:

```
127.0.0.1 - - [08/Nov/2023 16:29:02] "GET / HTTP/1.1" 404 -
 * Serving Flask app 'bokeh_app_fd60335a86254100a2a6631df1f58797'ents_source
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5050
Press CTRL+C to quit
```
i.e. the server just restarted (still 404). I hit CTRL+C again, and the following happens:

```
copying bokeh-plot files... [100%] bokeh-content-fbe67463385d4aabb3ee0fe6f09fd2c2-docs-examples-basic-bars-bar.js
adding links to sitemap...

Sphinx error:
Error generating bokeh-content-2f8ebb4ba27240b6be1c650f99705f86-docs-examples-basic-layouts-custom_layout.js:

bokeh-plot:: error:

Traceback (most recent call last):
  File "C:\github\bokeh\src\bokeh\application\handlers\code_runner.py", line 229, in run
    exec(self._code, module.__dict__)
  File "C:\github\bokeh\examples\basic\layouts\custom_layout.py", line 61, in <module>
    save([p0, p1, p2], template=template)
  File "C:\github\bokeh\src\bokeh\sphinxext\example_handler.py", line 89, in _add_root
    curdoc().add_root(obj)
  File "C:\github\bokeh\src\bokeh\document\document.py", line 321, in add_root
    with self.models.freeze():
  File "C:\Users\Timo Metzger\anaconda3\envs\bkdev\lib\contextlib.py", line 142, in __exit__
    next(self.gen)
  File "C:\github\bokeh\src\bokeh\document\models.py", line 135, in freeze
    self._pop_freeze()
  File "C:\github\bokeh\src\bokeh\document\models.py", line 288, in _pop_freeze
    self.recompute()
  File "C:\github\bokeh\src\bokeh\document\models.py", line 215, in recompute
    new_models |= mr.references()
AttributeError: 'list' object has no attribute 'references'


evaluating source:

from bokeh.io import save
from bokeh.plotting import figure
from bokeh.util.browser import view

template = """
{% block preamble %}
<style>
* { box-sizing: border-box; }
.plots { display: flex; flex-direction: row; width: 100%; }
.p { width: 33.3%; padding: 50px; }
.p:nth-child(1) { background-color: red; }
.p:nth-child(2) { background-color: green; }
.p:nth-child(3) { background-color: blue; }
</style>
{% endblock %}
{% block body %}
<body style="background-color: lightgray;">
    {{ self.inner_body() }}
</body>
{% endblock %}
{% block contents %}
<div>
<p>This example shows how different Bokeh Document roots may be embedded in custom
templates. The individual plots were embedded in divs using the embed macro:

<pre>
    &lt;div class="p"&gt;&#123;&#123; embed(roots.p0) &#125;&#125;&lt;/div&gt;
    &lt;div class="p"&gt;&#123;&#123; embed(roots.p1) &#125;&#125;&lt;/div&gt;
    &lt;div class="p"&gt;&#123;&#123; embed(roots.p2) &#125;&#125;&lt;/div&gt;
</pre>

And the divs are styled using standard CSS in the template:

<pre>
    .p { width: 33.3%; padding: 50px; }
    .p:nth-child(1) { background-color: red; }
    .p:nth-child(2) { background-color: green; }
    .p:nth-child(3) { background-color: blue; }
</pre>
</p>
</div>
<div class="plots">
    <div class="p">{{ embed(roots.p0) }}</div>
    <div class="p">{{ embed(roots.p1) }}</div>
    <div class="p">{{ embed(roots.p2) }}</div>
</div>
</div>
{% endblock %}
"""

x = [1, 2, 3]
y = [1, 2, 3]

p0 = figure(name="p0", sizing_mode="scale_width")
p0.scatter(x, y, size=20, fill_color="red")
p1 = figure(name="p1", sizing_mode="scale_width")
p1.scatter(x, y, size=20, fill_color="green")
p2 = figure(name="p2", sizing_mode="scale_width")
p2.scatter(x, y, size=20, fill_color="blue")

save([p0, p1, p2], template=template)
view("custom_layout.html")

File not found - static
0 File(s) copied
Terminate batch job (Y/N)? n
```

It seems like several strange things are happening here... 

I usually build the docs in WSL - I would expect `make.bat html` to work just like  `make html`  works in WSL!

**Comment by tcmetzger** [comment]
Closing this in favor of #13544

**Comment by mattpap** [comment]
The original problem was fixed in PR #13352.
